test/formatMessage.spec.js

```diff
@@ -0,0 +1,62 @@
+import { expect } from "chai";
+import React from "react";
+import { shallow } from "enzyme";
+import Globalize from "globalize";
+import FormatMessage from "../dist/message";
+
+Globalize.load(
+    require( "cldr-data/supplemental/plurals" ),
+    require( "cldr-data/supplemental/likelySubtags" )
+);
+
+Globalize.loadMessages({
+    en: {
+        salutations: {
+            hi: "Hi"
+        },
+        variables: {
+            hello: "Hello, {0} {1} {2}"
+        },
+        party: [
+            "{hostGender, select,",
+            "  female {{host} invites {guest} to her party}",
+            "    male {{host} invites {guest} to his party}",
+            "   other {{host} invites {guest} to their party}",
+            "}"
+        ],
+        task: [
+            "You have {count, plural,",
+            "     =0 {no tasks}",
+            "    one {one task}",
+            "  other {{formattedCount} tasks}",
+            "} remaining"
+        ]
+    }
+});
+
+describe("formatMessage Component", () => {
+    it("renders as a <span>", () => {
+        const wrapper = shallow(<FormatMessage locale="en" path="salutations/hi" />);
+        expect(wrapper.type()).to.eql("span");
+    });
+
+    it("resolves path and prints 'Hi'", () => {
+        const wrapper = shallow(<FormatMessage locale="en" path="salutations/hi" />);
+        expect(wrapper.text()).to.eql("Hi");
+    });
+
+    it("properly replaces variables", () => {
+        const wrapper = shallow(<FormatMessage locale="en" path="variables/hello" variables={["Wolfgang", "Amadeus", "Mozart"]} />);
+        expect(wrapper.text()).to.eql("Hello, Wolfgang Amadeus Mozart");
+    });
+
+    it("uses proper gender inflection", () => {
+        const wrapper = shallow(<FormatMessage locale="en" path="party" variables={{guest:"Mozart", guestGender:"male", host:"Beethoven", hostGender:"other"}} />);
+        expect(wrapper.text()).to.eql("Beethoven invites Mozart to their party");
+    });
+
+    it("uses proper plural inflection", () => {
+        const wrapper = shallow(<FormatMessage locale="en" path="task" variables={{count: 1}} />);
+        expect(wrapper.text()).to.eql("You have one task remaining");
+    });
+});

```

Comment:
This wasn't meant to be a comprehensive test suite but instead a start and we can add as we go. That said, I will add that test as it will fail during development of the solution for #34 which will be helpful